stdlib/REPL/src/REPL.jl

```diff
@@ -177,15 +177,17 @@ function check_for_missing_packages_and_run_hooks(ast)
     end
 end
 
-function modules_to_be_loaded(ast, mods = Symbol[])
+function modules_to_be_loaded(ast::Expr, mods = Symbol[])
     if ast.head in [:using, :import]
         for arg in ast.args
-            if first(arg.args) isa Symbol # i.e. `Foo`
-                if first(arg.args) != :. # don't include local imports
-                    push!(mods, first(arg.args))
+            arg = arg::Expr
+            arg1 = first(arg.args)
+            if arg1 isa Symbol # i.e. `Foo`
+                if arg1 != :. # don't include local imports
```

Comment:
Shouldn't really matter anymore since both types are now inferred, but perhaps also:
```suggestion
                if arg1 !== :. # don't include local imports
```

Reply:
Since it's now inferrable it's not necessary. I tend to use `===` and `!==` only when it makes a genuine difference.

Reply:
Yeah, makes sense.